errata_tool/erratum.py

```diff
@@ -275,7 +279,11 @@ def _fetch(self, errata_id):
             self.package_owner_email = advisory_old['people']['reporter']
             self.qe_email = advisory_old['people']['assigned_to']
             self.qe_group = advisory_old['people']['qe_group']
-            # self.manager_email = ???
+            self.manager_id = erratum.get('manager_id')
+            self.manager_email = erratum.get('manager_email')
+            if self.manager_email is None:
+                manager = User(self.manager_id)
```

Comment:
This will add an additional HTTP request inside the `Erratum._fetch()` method. Unfortunately that method already makes a lot of HTTP calls so it's already pretty slow.

Would you be ok removing this `User()` call from `_fetch()` and doing it outside the `Erratum` class inside your own code for now?

Ideally we will get a fix for this server-side, eventually.

Reply:
sure. Will do it.